frontend/src/scenes/surveys/SurveyRepeatSchedule.tsx

```diff
@@ -0,0 +1,130 @@
+                        {({ onChange, value }) => {
+                            return (
+                                <LemonInput
+                                    type="number"
+                                    data-attr="survey-iteration-count"
+                                    size="small"
+                                    min={1}
+                                    // NB this is enforced in the API too
+                                    max={500}
+                                    value={value || 1}
+                                    onChange={(newValue) => {
+                                        if (newValue && newValue > 0) {
+                                            onChange(newValue)
+                                        } else {
+                                            onChange(null)
+                                        }
+                                    }}
+                                    className="w-16"
+                                />
+                            )
+                        }}
+                    </LemonField>{' '}
+                    times, once every
+                    <LemonField name="iteration_frequency_days">
+                        {({ onChange, value }) => {
+                            return (
+                                <LemonInput
+                                    type="number"
+                                    data-attr="survey-iteration-frequency-days"
+                                    size="small"
+                                    min={1}
+                                    value={value || 90}
+                                    onChange={(newValue) => {
+                                        if (newValue && newValue > 0) {
+                                            onChange(newValue)
+                                        } else {
+                                            onChange(null)
+                                        }
+                                    }}
+                                    className="w-16"
+                                />
+                            )
+                        }}
+                    </LemonField>{' '}
+                    days
+                </div>
+            )}
+        </>
+    )
+}
+
+export function SurveyRepeatSchedule(): JSX.Element {
+    const { survey } = useValues(surveyLogic)
+    const { setSelectedSection } = useActions(surveyLogic)
+
+    const canSurveyBeRepeated = survey.conditions?.events?.repeatedActivation
+
+    return (
+        <div className="mt-4">
+            <h3> How often should we show this survey? </h3>
+            {canSurveyBeRepeated ? (
+                <span className="font-medium">
+                    <IconInfo className="mr-0.5" /> This survey is displayed whenever the event{' '}
+                    <LemonSnack>{survey.conditions?.events?.values.map((v) => v.name).join(', ')}</LemonSnack> is
```

Comment:
can it be that `survey.conditions?.events?.values` is null/empty? since we only check the `repeatedActivation` above and not the `values`.
Also, I think it makes sense that we do '{name of events} event(s)', so the name of the event(s) is before the word events.
like 'charge event' instead of 'event charge'

Reply:
fixed, and also now considering the plurality too:

<img width="786" alt="image" src="https://github.com/user-attachments/assets/31e935cc-f9ce-45c1-b49c-04ea5a8400c2" />